marklogic-data-hub-central/ui/src/components/entities/create-edit-step-dialog/create-edit-step-dialog.test.tsx

```diff
@@ -46,13 +47,13 @@ describe('Create Edit Step Dialog component', () => {
     expect(getByLabelText('Collection')).toBeInTheDocument();
     expect(getByLabelText('Query')).toBeInTheDocument();
     expect(getByLabelText('collection-input')).toBeInTheDocument();
-    expect(getByText('Save')).toBeDisabled();
+    expect(getByText('Save')).toBeEnabled();
     expect(getByText('Cancel')).toBeEnabled();
     //Collection radio button should be selected by default
     expect(getByLabelText('Collection')).toBeChecked();
   });
 
-    test('Verify merging name is mandatory and Save button is disabled', () => {
+    test('Verify save button is always enabled', () => {
```

Comment:
Can we add an enabled check on the Save button upon load of the form (line 60)? It's slightly different from the last case in the test in that the the required field (name) has never been touched: `expect(saveButton).toBeEnabled();`

Reply:
added